components/header.tsx

```diff
@@ -45,6 +70,32 @@ export default function Header() {
           <NavLink p={2}>News</NavLink>
         </Link>
       </Flex>
+      <Button
+        className="burger-menu"
+        onClick={toggleMenu}
+        sx={{
+          position: 'absolute',
+          cursor: 'pointer',
+          right: 20,
+          top: 20,
+          appearance: 'none',
+          display: 'inline-block',
+          textAlign: 'center',
+          lineHeight: 'inherit',
+          textDecoration: 'none',
+          p: 2,
+          backgroundColor: t => t.colors.mutedTransparent,
+          ':hover': {
+            backgroundColor: t => t.colors.mutedTransparent,
+          },
+          [MEDIA_QUERY.d('min', 768)]: {
```

Comment:
Also something I forgot, in theme-ui you have a better syntax to handle media queries.
I know it might be confusing at first but you can set properties using array
https://theme-ui.com/sx-prop#responsive-values
In the theme I have defined 3 endpoints already. So everytime you set something as array, you will specify first value for mobile, second for tabled and third for desktop.
eg
```
margin: [0, 2, 5] // for margin you can use m: [0, 2, 5]
```
That would mean. Margin 0 for mobile, 2 for tablet and 5 for desktop. Also when you define those numbers you are not referring to px but to values defined in the theme, but anyway you get the point.

In this specific case I think you can do this:
```
display: ["block", "none"]
```
In this case I'm not defining the third value for my array because second and third will be the same, so you can omit the third one.